synapse/lib/stormlib/stix.py

```diff
@@ -653,10 +653,12 @@ def getObjLocals(self):
             'validate': self.validateBundle,
         }
 
+    @s_stormtypes.stormfunc(readonly=True)
```

Comment:
The semafork is limited to a fixed number of forks, right? Could this be used to fork bomb?

Reply:
fork bomb; no. We limit the # of concurrent semafork calls such that the forkpool always has reserved workers for parsing storm queries.

Bad input here ( bad meaning the stix validation to go out to lunch ) would be a DOS regardless of readonly status.

Reply:
Per discussion we'll leave this as is ( with the readonly=true marking ).